src/msg.c

```diff
@@ -1364,6 +1314,33 @@ static bool handle_device_removed(const char *device_id)
 	return false;
 }
 
+static bool handle_schema_updated(struct session *session,
+				  const char *device_id, const char *err)
+{
+	struct knot_device *device;
+
+	if (err) {
+		hal_log_error("%s", err);
```

Comment:
Send the following code to thing `KNOT_ERR_CLOUD_FAILURE` to avoid it to keep waiting for the schema response.

Reply:
Done.

Reply:
Put a TODO message in this commit 0bdd315a94e8280d119a1fe11db284e385b7fb51 telling to send an error to thing

Reply:
It will inform the Thing about an error when sends a KNOT_MSG_SCHM_END_RSP.